app/views/LoginSignupView.js

```diff
@@ -329,16 +337,29 @@ class LoginSignupView extends React.Component {
 				onPress = () => this.onPressCustomOAuth(service);
 				break;
 			}
+			case 'saml': {
+				onPress = () => this.onPressSaml(service);
+				break;
+			}
 			default:
 				break;
 		}
+		name = name.charAt(0).toUpperCase() + name.slice(1);
+		let buttonText;
+		if (service.service === 'saml') {
+			buttonText = <Text style={styles.serviceName}>{name}</Text>;
+		} else {
+			buttonText = (
+				<>
```

Comment:
`<React.Fragment>`?

Reply:
@diegolmello I think it is related to your changes ˆˆ

Reply:
@djorkaeffalexandre this is a shorthand for `<React.Fragment>` :)

Reply:
Nice! 🙀